EnglishFilter/sections/whitelist.txt

```diff
@@ -214,6 +214,8 @@ copypastecharacter.com#$##contents { visibility: visible!important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/55620
 @@||edge.microsoft.com/translate/auth
 @@||api.cognitive.microsofttranslator.com/translate?
+! https://github.com/AdguardTeam/AdguardFilters/issues/54041
+@@||*thalia.de*&utm_source=www.brickmerge.de
```

Comment:
I see three URLs with `&banner_id=`
```
https://t.myvisualiq.net/click_pixel?*&red=https://www.thalia.de/shop/*&banner_id=*
https://t.myvisualiq.net/ul_cb/click_pixel?*&red=https://www.thalia.de/shop/*&banner_id=*
https://thalia.de/shop/*.html?pid=*&group_id=0&banner_id=
```
What exactly must be excluded?

Reply:
@Sergey-Lyapin

Reply:
It seems that such a rule is enough
```
!+ PLATFORM(ios, ext_safari)
@@||*thalia.de/shop/*.html?pid=*&group_id=0&banner_id=
```

Reply:
Why not just `@@thalia.de/shop/...`?

Reply:
Yes, it looks better